CONTRIBUTING.md

```diff
@@ -150,3 +152,20 @@ Follow GitHub's fork-branch-pull request pattern.
 4. The project team will label the issue and evaluate the pull request in the weekly configuration control board (CCB) meeting. For more information, visit [The cFS CCB Process.](https://github.com/nasa/cFS/wiki/The-cFS-CCB-Process)
 5. If the pull request is accepted, it will be merged into cFS.
 
+## Writing High-Quality Code
+
+1. Follow cFS code conventions (formatting, symbol naming, file naming, etc). Do *not* change/reformat existing code, except to address your changes.
```

Comment:
Let's add something about the clang-format workflow in each submodule. For the symbol naming etc., I think there's a section in the users guide so it might be good to link to it here as well

Reply:
I couldn't find information in the cFE and OSAL users guide for guidance on symbol naming and other code convection topics.

Reply:
Added a link to the cFS_IdentifierNamingConvention document as well.

Reply:
great!